packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-matrix.stories.ts

```diff
@@ -149,10 +160,21 @@ export const richTextEditorSizing: StoryFn = createStory(html`
     ])}
 `);
 
+// prettier-ignore
 const mobileWidthComponent = html`
-    <${richTextEditorTag} style="padding: 20px; width: 300px; height: 250px;">
-        <${buttonTag} slot="footer-actions" appearance="ghost">Cancel</${buttonTag}>
-        <${buttonTag} slot="footer-actions" appearance="outline">Ok</${buttonTag}>
+    <${richTextEditorTag} style="padding: 20px; width: 360px; height: 250px;">
+        <${richTextMentionUsersTag} pattern="^user:(.*)">
+            <${mappingUserTag} key="user:1" display-name="John Doe"></${mappingUserTag}>
+        </${richTextMentionUsersTag}>
+        <${buttonTag}
+            style="height: var(${cssPropertyFromTokenName(tokenNames.controlSlimHeight)}); width: 72px;"
```

Comment:
Instead of `cssPropertyFromTokenName(tokenNames.controlSlimHeight)`, use `controlSlimHeight.cssCustomProperty` (in this file and the other stories file).